src/main/java/com/taskadapter/redmineapi/internal/Transport.java

```diff
@@ -602,6 +606,19 @@ public void setPassword(String password) {
 	public void setLogin(String login) {
 		setCredentials(login, password);
 	}
+        /**
+         * @return the impersonateUser
+         */
+        public String getImpersonateUser() {
+           return impersonateUser;
+        }
+
+        /**
+        * @param impersonateUser the impersonateUser to set
+        */
```

Comment:
this javadoc does not explain anything. what is this user for? use-cases? how to use it?